Core/GDCore/IDE/Events/ExpressionCompletionFinder.h

```diff
@@ -182,16 +260,18 @@ class GD_CORE_API ExpressionCompletionFinder
   static std::vector<ExpressionCompletionDescription>
   GetCompletionDescriptionsFor(gd::ExpressionNode& node,
                                size_t searchedPosition) {
-    gd::ExpressionNode* nodeAtLocation =
-        gd::ExpressionNodeLocationFinder::GetNodeAtPosition(node,
-                                                            searchedPosition);
-
+    
+    gd::ExpressionNodeLocationFinder finder(searchedPosition);
+    node.Visit(finder);
+    gd::ExpressionNode* nodeAtLocation = finder.GetNode();
+    gd::ExpressionNode* parentNodeAtLocation = finder.GetParentNode();
+    
     if (nodeAtLocation == nullptr) {
       std::vector<ExpressionCompletionDescription> emptyCompletions;
       return emptyCompletions;
     }
 
-    gd::ExpressionCompletionFinder autocompletionProvider(searchedPosition);
+    gd::ExpressionCompletionFinder autocompletionProvider(searchedPosition, *parentNodeAtLocation);
```

Comment:
This is dereferencing a pointer that can (and is) null. I'm fixing this, as this can generate horrible crashs if used.